src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionSelectItem.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { ProposedSolution } from 'shared';
+import { Badge, Col, Container, Row, Card } from 'react-bootstrap';
+import { dollarsPipe, weeksPipe } from '../../utils/Pipes';
+
+interface ProposedSolutionSelectItemProps {
+  proposedSolution: ProposedSolution;
+  selected: boolean;
+  onClick: () => void;
+}
+
+const ProposedSolutionSelectItem: React.FC<ProposedSolutionSelectItemProps> = ({
+  proposedSolution,
+  selected,
+  onClick: setter
+}) => {
+  const spacer = 'mb-1';
+
+  const component = (
+    <Card className={'mb-3'} bg={'light'} border={'dark'}>
+      <Card.Body>
+        <Container onClick={setter} fluid>
+          <Row className={spacer}>
+            <b>Description</b>
+            {selected ? (
+              <Badge
+                className={'mr-3'}
+                variant="success"
+                style={{ marginLeft: '66%', textAlign: 'right' }}
```

Comment:
because the styling is more than 1 thing ( and you have a similar situation below), I would break this out into a stylesheet or make a style object in this file and call it here